### PR TITLE
Search screen

### PR DESCRIPTION
**Pull request by felire**
### Summary
I've created the search screen, to do this i use redux to associate the header component with the screen.

### Gifs

transition3

### CODE REVIEW

File: WBooks/src/app/screens/Search/components/IsEmptyBookList/index.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { View, Text } from 'react-native';
+
+import styles from './styles';
+
+function IsEmptyInputText() {
+  return (
+    <View style={styles.container}>
+      <View>
```
Comment (santinucera) [inline WBooks/src/app/screens/Search/components/IsEmptyBookList/index.js]: Use fragment

File: WBooks/src/app/screens/Search/components/IsEmptyInputText/index.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import styles from './styles';
+
+function IsEmptyInputText() {
+  return (
+    <View style={styles.container}>
+      <Image source={searchIcon} style={styles.image} />
+      <View>
+        <Text style={styles.title}>Search in WBooks</Text>
+      </View>
+      <View>
```
Comment (santinucera) [inline WBooks/src/app/screens/Search/components/IsEmptyInputText/index.js]: Fragment

File: WBooks/src/redux/search/reducer.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import Immutable from 'seamless-immutable';
+
+import { actionTypes } from './constants/actionTypes';
+
+const initialState = {
+  text: '',
+  books: []
+};
+
+export const searchReducer = (state = Immutable(initialState), action) => {
```
Comment (santinucera) [inline WBooks/src/redux/search/reducer.js]: Why don't you use redux recompose?

Comment (felire) [reply WBooks/src/redux/search/reducer.js]: It's a very simple action without much logic. And there is not failure, get or success actions

**Pull request merged**